src/SFA.DAS.ApplyService.Application/Apply/UpdatePageAnswers/UpdatePageAnswersHandler.cs

```diff
@@ -4,285 +4,251 @@
+                    pageAnswers.Answers.Add(existingAnswer);
+                }
+                else
+                {
+                    var validators = _validatorFactory.Build(question);
+                    foreach (var validator in validators)
+                    {
+                        var errors = validator.Validate(question, answer);
 
-            if (!errors.Any()) continue;
+                        if (!errors.Any()) continue;
 
-            validationPassed = false;
-            validationErrors.AddRange(errors);
-          }
+                        validationPassed = false;
+                        validationErrors.AddRange(errors);
+                    }
 
-          if (answer != null)
-          {
-            pageAnswers.Answers.Add(answer);  
-          }
-        }
-      }
-      else
-      {
-        var answer = request.Answers.FirstOrDefault(a => a.QuestionId == question.QuestionId);
-
-        var validators = _validatorFactory.Build(question);
-        foreach (var validator in validators)
-        {
-          var errors = validator.Validate(question, answer);
-
-          if (errors.Any())
-          {
-            validationPassed = false;
-            validationErrors.AddRange(errors);
-          }
-          else
-          {
-            if (question.Input.Type == "Checkbox" && answer.Value == "on")
+                    if (answer != null)
+                    {
+                        pageAnswers.Answers.Add(answer);
+                    }
+                }
+            }
+            else
             {
-              answer.Value = "Yes";
+                var validators = _validatorFactory.Build(question);
+                foreach (var validator in validators)
+                {
+                    var errors = validator.Validate(question, answer);
+
+                    if (errors.Any())
+                    {
+                        validationPassed = false;
+                        validationErrors.AddRange(errors);
+                    }
+                    else
+                    {
+                        if (question.Input.Type == "Checkbox" && answer.Value == "on")
+                        {
+                            answer.Value = "Yes";
+                        }
+                    }
+                }
+
+                if (answer != null)
+                {
+                    pageAnswers.Answers.Add(answer);
+                }
             }
-          }
+
+            return validationPassed;
         }
 
-        if (answer != null)
+        private static bool ExistingUpload(string questionId, List<PageOfAnswers> pagePageOfAnswers)
         {
-          pageAnswers.Answers.Add(answer);
+            return pagePageOfAnswers.Any(poa => poa.Answers.Any(a => a.QuestionId == questionId && a.Value != ""));
         }
-      }
-
-      return validationPassed;
     }
```

Comment:
Not entirely sure what happened with this file when editing, but it makes it extremely hard to see what's changed.

Reply:
This file was formatted with 2 space tabs; and all the others in the project with 4 space tabs so I reformatted the file - this worked fine in visual studio and the differ showed only the changes but it has totally thrown github - I'll try and resolve this

Reply:
Using the Diff settings drop down select 'Hide whitespace changes' - there is still some oddities as line move inwards to 4 space tabs - but this is acceptable given the file has been reformatted to match the others in the project.